### DISCUSSION TITLE
project not able to run on localhost

### DISCUSSION THREAD

**Discussion by AnushkaS77** [code-a2z/jarvis#185] [Q&A] [answered]
to run the project, as streamlit is required, even after running the required pip command streamlit is not yet installed.
i tried to uninstall and reinstall the latest ver of python as...along with adding path to environment variables. Yet the same error.
![11](https://github.com/user-attachments/assets/ac4493e2-7305-496e-b081-9c35fa7b1f09)

**Comment by Avdhesh-Varshney** [comment, marked as answer]
Have you tried,

```python
pip install -r requirements.txt
```

Try to install it manually,
```python
pip install streamlit
```

There is 1 mistake to run the Application,
```python
streamlit run Jarvis.py
```

Read these guidelines properly [link](https://github.com/Avdhesh-Varshney/Jarvis/blob/main/Contributing.md#creating-first-pull-request-)

**Comment by Zedoman** [comment]
@AnushkaS77 have u solved the issue?

**Reply by AnushkaS77** [reply]
yes im working on it ...would create a PR asap

**Comment by AnushkaS77** [comment]
![Screenshot 2024-10-17 221907](https://github.com/user-attachments/assets/961607d3-c005-4f6a-8933-3961460ecf95)
is it necessary to download pyarrow...its not mentioned in the requirements.txt ? as its the only error im getting

**Comment by Zedoman** [comment]
ya install it to remove the error
